Minor fixes to spelling and grammar (#98)
se-edu/se-book: revisionControl/what/q-essay-rcs-explain.md

<panel header="{{ icon_Q }} Explain RCS in a team project context">
<question has-input="true">

> Suppose you are doing a team project with Tom, Dick, and Harry but those three have not even heard the term RCS. How do you explain RCS to them as briefly as possible, using the project as an example?

Reviewer: ```suggestion
Suppose you are doing a team project with Tom, Dick, and Harry, all of whom have not even heard of the term RCS. How do you explain RCS to them as briefly as possible, using the project as an example?
```
